src/server.ts

```diff
@@ -133,6 +134,12 @@ wss.on('connection', ws => {
         data.configData,
       );
     }
+    else if (data.type === 'createSemanticLayer') {
```

Comment:
We can leave this for now, will activate it in the future when Nazim is done with the semantic layer work